pypowsybl/network.py

```diff
@@ -4124,6 +4124,228 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def create_load_bay(self, busbar_section_id: str, load_position_order: int, df: _DataFrame,
+                        load_direction: str = "BOTTOM", throw_exception: bool = False,
+                        reporter: _Reporter = None, **kwargs: _ArrayLike) -> None:
+        """
+        Creates a load, connects it to the network and creates the topology. The voltage level containing the
```

Comment:
In all docstrings:
we must have a short description as the first line, because it's used in the HTML doc as a summary where there is not much space (in table cells).